v2/pkg/apis/kubeflow/validation/validation.go

```diff
@@ -46,6 +46,9 @@ func validateMPIJobSpec(spec *kubeflow.MPIJobSpec, path *field.Path) field.Error
 	} else if !validCleanPolicies.Has(string(*spec.CleanPodPolicy)) {
 		errs = append(errs, field.NotSupported(path.Child("cleanPodPolicy"), *spec.CleanPodPolicy, validCleanPolicies.List()))
 	}
+	if spec.SSHAuthMountPath == "" {
```

Comment:
does this validation contradict with the [defaulting here](https://github.com/kubeflow/mpi-operator/pull/383/files#diff-e4f5a918268e7d4cc61f43c32e525cfb9ff706c0a57ecc0372a2259b22d44c09R59)?